packages/client/tests/e2e/_utils/standard.cmd.sh

```diff
@@ -1,19 +1,35 @@
-#!/bin/sh
+#!/bin/bash
 
+# Database Env Vars
+export PRISMA_DB_NAME=$(cat /dev/urandom | tr -dc 'a-z0-9' | head -c 8)
+export TEST_E2E_POSTGRES_URI="postgres://prisma:prisma@172.17.0.1:5432/${PRISMA_DB_NAME}"
```

Comment:
`172.17.0.1` is the default docker bridge address, to easily access the dbs exposed on localhost